bot.py

```diff
@@ -32,7 +32,7 @@ def path_exists(filename):
 
     with open("config.json", "a") as config_file:
         config_file.write(
-            json.dumps({"nickname": bot_name, "prefix": prefix, "channels": [channel]})
+            json.dumps({"nickname": bot_name, "prefix": prefix, "channels": [channel], "owner_id": []})
```

Comment:
Use the same logic as what you did in `setup.py` and add another question for the person’s ID.